src/ngraph/frontend/onnx_import/utils/convpool.hpp

```diff
@@ -33,19 +33,44 @@ namespace ngraph
             /// \return The kernel Shape object representing its dimensions (height, width, depth).
             Shape get_kernel_shape(const Node& node);
 
-            /// \brief  Get number of pixels to stride operation by in each direction.
             ///
-            /// \param node The Node ptr representing Conv or Pool operation.
-            /// \return The kernel Shape object representing its dimensions (height, width, depth).
+            /// \brief      Get number of pixels to stride operation by in each direction.
+            ///
+            /// \param      node  The Node ptr representing Conv or Pool operation.
+            ///
+            /// \return     The kernel Shape object representing its dimensions (height, width,
+            ///             depth).
             Strides get_strides(const Node& node);
 
-            /// \brief Get number of pixels for filter dilation in each direction.
             ///
-            /// \param node The Node ptr representing ONNX operation.
-            /// \return The Strides object containing number of pixels for filter dilation
-            ///         (height, width, depth).
+            /// \brief      Get number of pixels to stride operation by in each direction.
+            ///
+            /// \param      node         The Node ptr representing Conv or Pool operation.
+            /// \param[in]  kernel_rank  The operator's kernel rank.
+            ///
+            /// \return     The kernel Shape object representing its dimensions (height, width,
+            ///             depth).
+            Strides get_strides(const Node& node, const std::size_t kernel_rank);
+
+            ///
+            /// \brief      Get number of pixels for filter dilation in each direction.
+            ///
+            /// \param      node  The Node ptr representing ONNX operation.
+            ///
+            /// \return     The Strides object containing number of pixels for filter dilation
+            ///             (height, width, depth).
             Strides get_dilations(const Node& node);
 
+            ///
+            /// \brief      Get number of pixels for filter dilation in each direction.
+            ///
+            /// \param      node         The Node ptr representing ONNX operation.
+            /// \param[in]  kernel_rank  The operator'skernel rank.
+            ///
+            /// \return     The Strides object containing number of pixels for filter dilation
+            ///             (height, width, depth).
+            Strides get_dilations(const Node& node, const std::size_t kernel_rank);
+
```

Comment:
As above
```suggestion
            ///
            /// \brief      Get number of pixels for filter dilation in each direction.
            ///
            /// \param      node         The Node ptr representing ONNX operation.
            /// \param[in]  kernel_rank  The operator'skernel rank.
            ///
            /// \return     The Strides object containing number of pixels for filter dilation
            ///             (height, width, depth).
            Strides get_dilations(const Node& node, const std::size_t kernel_rank = 0);

```
Please be consistent with the `[in]` part of the param.